notes/20200217_meeting.md

```diff
@@ -20,7 +20,6 @@ We agreed that the biggest benefits of RAP are:
 
 Other benefits of RAP include: 
 - Better quality assurance (e.g. by standardising practices)
-- Improved timeliness
```

Comment:
Just wondering about your reasons for removing this? It was definitely something the group felt strongly about if I remember rightly. It could probably be expanded though because I think it means both a quicker end-to-end process as well as potentially sharing more timely information with users.

Reply:
I definitely don't disagree with the point, I just felt it was already captured under efficiency.

Reply:
Ah fair enough, thanks! I think it might be a slightly extended version of efficiency but we can capture that somehow.